Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN apt-get install -y build-essential \
     python-zmq
 
 RUN pip install py4j \
-    ipython[notebook] \
+    ipython[notebook]==3.2 \
```

Comment:
latest version is fine here. Do you have any bug report to prove v4.0 is not supports profile correctly?